src/modal/modal.component.ts

```diff
@@ -128,6 +128,11 @@ export class Modal implements AfterViewInit, OnInit, OnDestroy {
 	 */
 	@Input() modalLabel = "default";
 
+	/**
+	 * Determins if the modal closes on an overlay click.
+	 */
+	@Input() onOverlayClick: "default" | "override" = "default";
```

Comment:
Modal doesn't need this, since the user can choose how to handle `overlaySelected` outside of the component